packages/spor-accordion-react-native/src/Expandable.tsx

```diff
@@ -71,7 +73,7 @@ export const Expandable = ({
   }
 
   return (
-    <Box style={style as any} ml="sm" mb="sm">
+    <Box style={[restyleStyle as any, style]}>
       <Pressable
         style={isPressed ? pressedStyle : { padding: 12 }}
```

Comment:
Blir dette riktig mon tro? 
```suggestion
        style={isPressed ? pressedStyle : { padding: theme.size.spacing[2] }}
```